src/modules/Lan.rb

```diff
@@ -876,14 +877,14 @@ def activate_network_service
         NetworkService.ReloadOrRestart if Stage.normal || !Linuxrc.usessh
 
       when :remote_installer
-        ifaces = LanItems.getNetworkInterfaces
+        connections = yast_config&.connections&.map(&:name) || []
```

Comment:
NP: connections name sounds to be a bit misleading here, I'd keep ifaces or names or something like that

Reply:
@mchf updated